package.json

```diff
@@ -127,7 +129,7 @@
     }
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": ">= 14.*"
```

Comment:
Running `ember s` or `ember t` resulted in an error (this happened even in the `main` branch at the time of writing):

```sh
building... [Babel: ember-test-helpers > applyPatches]node:internal/validators:164
      throw new ERR_INVALID_ARG_TYPE(name, 'Object', value);
      ^

TypeError [ERR_INVALID_ARG_TYPE]: The "options" argument must be of type object. Received an instance of Array
    at ChildProcess.target.send (node:internal/child_process:733:7)
    at Array.forEach (<anonymous>)
    at dispatchQueuedRequests (/ember-intl/node_modules/@ember/test-helpers/node_modules/workerpool/lib/WorkerHandler.js:174:21)
    at ChildProcess.<anonymous> (/ember-intl/node_modules/@ember/test-helpers/node_modules/workerpool/lib/WorkerHandler.js:129:7)
    at ChildProcess.emit (node:events:527:28)
    at ChildProcess.emit (node:domain:475:12)
    at emit (node:internal/child_process:938:14)
    at processTicksAndRejections (node:internal/process/task_queues:84:21) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

According to https://github.com/vercel/vercel/pull/7782, we'll want to pin Node to `14`.

Reply:
I think we should say so in the release notes, but it wouldn't necessarily fail the install, unless [`engine-strict`](https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict) is set.

Given that `10` and `12` have long reached their EOL, I think it's fine to drop them.